.github/CODEOWNERS

```diff
@@ -0,0 +1,8 @@
+# Default owner is the maintainer team
+* @ppb/maintainers
+
+# @nbraud owns the Hypothesis testsuite
```

Comment:
I like this experiment. If it works here in Vector, I will probably do some of this regarding bits of the core engine.